### ISSUE TITLE
 AwasomeTTS speaks text twice when reviewing cards.

### ISSUE THREAD

**Issue by eugen-nesterov** [AwesomeTTS/awesometts-anki-addon#255]
When I try to use this code 
`{{tts en_US voices=AwesomeTTS:Answer}}`
to speak the answer it speaks the text once when editing/previewing a card, and, for some reason, twice when reviewing. Is it a bug, is there any way to fix it and prevent speaking the text twice?

https://user-images.githubusercontent.com/87163666/161663042-4d3d785e-d676-4221-9817-240ab76eafd5.mp4

**Comment by luc-languagetools** [comment]
can you share your card templates ?

**Comment by eugen-nesterov** [comment]
Sure.

Back Template:

`<div class="text">
	Голицынский Ю.Б – Грамматика
	<br>
	АРТИКЛЬ
	<br>
	{{Exercise number}}
	<br>
	{{Exercise task}}
</div>


<div class="divider"></div>

<div class="text">
	{{cloze:To solve}}
	{{type:cloze:To solve}}
</div>

{{tts en_US voices=AwesomeTTS:Answer}}

<div class="divider"></div>

<div id="myButton" style="margin: 1rem 1rem; text-align: center">
	<button onclick="myFunction()">Показать правила</button>
</div>

<div id="hint" style="display: none">

<div class="text">
	{{edit:Hint}}
</div>

</div>

 <script>

	function myFunction() {
	var x = document.getElementById("hint");
	
	if (x.style.display === "none") {
		x.style.display = "block";
		document.getElementById("myButton").style.display = "none";

	} else {
		x.style.display = "none";
	}
}

</script>`

Fields:
<img width="168" alt="Screen Shot 2022-04-05 at 05 13 40" src="https://user-images.githubusercontent.com/87163666/161665695-85db7944-748a-4a96-835d-ed934c020671.png">

Card example:
<img width="800" alt="Screen Shot 2022-04-05 at 05 15 04" src="https://user-images.githubusercontent.com/87163666/161665847-7e185c1b-4ccd-4e61-90d8-9ee36222766f.png">

Back Preview:
<img width="551" alt="Screen Shot 2022-04-05 at 05 15 59" src="https://user-images.githubusercontent.com/87163666/161665943-a6b668b1-5972-4d5c-9f1f-ce40f48ef44c.png">

**Comment by luc-languagetools** [comment]
From the card template, i don't see anything wrong, so it's not clear to me why you're hearing the audio twice. You could try debug logging to file if you understand how to set it up: https://github.com/AwesomeTTS/awesometts-anki-addon/wiki/Debug-Logging
is there something preventing you from upgrading to the latest AwesomeTTS?

**Comment by eugen-nesterov** [comment]
@luc-languagetools, I have just updated it to the latest version 1.60 but the problem still remains.

https://user-images.githubusercontent.com/87163666/161672613-ed1aff7a-8de7-42bb-8289-5ad0659b9548.mp4

Nevertheless, thanks for your attention and replies.

**Comment by luc-languagetools** [comment]
what about the debugging logging method I suggested, are you able to try that ?

**Comment by eugen-nesterov** [comment]
@luc-languagetools Nothing happens. Am I doing something wrong? Is there a comprehensive tutorial about using debugging logging method

https://user-images.githubusercontent.com/87163666/161676774-9fe35430-9fb2-44b1-b58f-faeef513f340.mp4

?

**Comment by eugen-nesterov** [comment]
@luc-languagetools What command should I use and what kind of information should I retrieve? Is there a comprehensive guide to using the debugging logging feature?
<img width="571" alt="Screen Shot 2022-04-05 at 07 15 29" src="https://user-images.githubusercontent.com/87163666/161677858-b402090e-69e7-4374-b063-71e40eb982a1.png">

**Comment by luc-languagetools** [comment]
can you try following these instructions ? https://github.com/AwesomeTTS/awesometts-anki-addon/wiki/Debug-Logging

**Comment by eugen-nesterov** [comment]
Yes, I did. What should I do next? What commands should I enter next?
<img width="697" alt="Screen Shot 2022-04-05 at 08 08 46" src="https://user-images.githubusercontent.com/87163666/161683097-3200ead9-e7b0-4cf0-ae81-dfd190d2aa96.png">

**Comment by luc-languagetools** [comment]
here i'm a bit less familiar, are you attempting to run the Anki command ? Does the Anki application start up ? And did you set the require environment variable ?
I'm planning on getting a hold of a mac soon to try to make this whole process easier.
